lib/runtime-core/src/state.rs

```diff
@@ -1022,62 +1111,101 @@ pub mod x64 {
         unreachable!();
     }
 
+    /// A kind of GPR register
     #[repr(u8)]
     #[derive(Copy, Clone, Debug, Eq, PartialEq, Ord, PartialOrd, Hash)]
     pub enum GPR {
+        /// RAX Register
```

Comment:
Is there a way to suppress the extra comments here? These aren't adding value, and they're doubling the number of lines.

Reply:
@nlewycky They can be suppressed individually using `#[doc(hidden)]` but that would be the same number of lines. I think they would all be suppressed with `#[doc(hidden)]` on the enum but then we wouldn't get a doc for that.